[2023-09-18T13:47:24.638+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-16T00:00:00+00:00 [queued]>
[2023-09-18T13:47:24.649+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-16T00:00:00+00:00 [queued]>
[2023-09-18T13:47:24.650+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-09-18T13:47:24.672+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): task2> on 2023-09-16 00:00:00+00:00
[2023-09-18T13:47:24.678+0000] {standard_task_runner.py:57} INFO - Started process 4488 to run task
[2023-09-18T13:47:24.683+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '6.1-monitoring', 'task2', 'scheduled__2023-09-16T00:00:00+00:00', '--job-id', '262', '--raw', '--subdir', 'DAGS_FOLDER/6.1-monitoring.py', '--cfg-path', '/tmp/tmpzx4pjilu']
[2023-09-18T13:47:24.686+0000] {standard_task_runner.py:85} INFO - Job 262: Subtask task2
[2023-09-18T13:47:24.747+0000] {task_command.py:415} INFO - Running <TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-16T00:00:00+00:00 [running]> on host e891a57bf0af
[2023-09-18T13:47:24.847+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='6.1-monitoring' AIRFLOW_CTX_TASK_ID='task2' AIRFLOW_CTX_EXECUTION_DATE='2023-09-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-16T00:00:00+00:00'
[2023-09-18T13:47:24.848+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-18T13:47:24.850+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "sleep 2 && echo 'Task 2"]
[2023-09-18T13:47:24.865+0000] {subprocess.py:86} INFO - Output:
[2023-09-18T13:47:24.867+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: unexpected EOF while looking for matching `''
[2023-09-18T13:47:24.869+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 2: syntax error: unexpected end of file
[2023-09-18T13:47:24.871+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-09-18T13:47:24.885+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-09-18T13:47:24.890+0000] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=6.1-monitoring, task_id=task2, execution_date=20230916T000000, start_date=20230918T134724, end_date=20230918T134724
[2023-09-18T13:47:24.910+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 262 for task task2 (Bash command failed. The command returned a non-zero exit code 2.; 4488)
[2023-09-18T13:47:24.936+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-09-18T13:47:24.966+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-18T13:53:34.172+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-16T00:00:00+00:00 [queued]>
[2023-09-18T13:53:34.264+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-16T00:00:00+00:00 [queued]>
[2023-09-18T13:53:34.265+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-09-18T13:53:34.284+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): task2> on 2023-09-16 00:00:00+00:00
[2023-09-18T13:53:34.290+0000] {standard_task_runner.py:57} INFO - Started process 5136 to run task
[2023-09-18T13:53:34.297+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '6.1-monitoring', 'task2', 'scheduled__2023-09-16T00:00:00+00:00', '--job-id', '419', '--raw', '--subdir', 'DAGS_FOLDER/6.1-monitoring.py', '--cfg-path', '/tmp/tmp20tjdwvi']
[2023-09-18T13:53:34.300+0000] {standard_task_runner.py:85} INFO - Job 419: Subtask task2
[2023-09-18T13:53:34.366+0000] {task_command.py:415} INFO - Running <TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-16T00:00:00+00:00 [running]> on host e891a57bf0af
[2023-09-18T13:53:34.484+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='6.1-monitoring' AIRFLOW_CTX_TASK_ID='task2' AIRFLOW_CTX_EXECUTION_DATE='2023-09-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-16T00:00:00+00:00'
[2023-09-18T13:53:34.486+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-18T13:53:34.488+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "sleep 2 && echo 'Task 2'"]
[2023-09-18T13:53:34.502+0000] {subprocess.py:86} INFO - Output:
[2023-09-18T13:53:36.506+0000] {subprocess.py:93} INFO - Task 2
[2023-09-18T13:53:36.509+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-09-18T13:53:36.556+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=6.1-monitoring, task_id=task2, execution_date=20230916T000000, start_date=20230918T135334, end_date=20230918T135336
[2023-09-18T13:53:36.606+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-18T13:53:36.651+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-09-18T14:06:25.992+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-16T00:00:00+00:00 [queued]>
[2023-09-18T14:06:26.022+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-16T00:00:00+00:00 [queued]>
[2023-09-18T14:06:26.027+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-09-18T14:06:26.058+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): task2> on 2023-09-16 00:00:00+00:00
[2023-09-18T14:06:26.069+0000] {standard_task_runner.py:57} INFO - Started process 5580 to run task
[2023-09-18T14:06:26.081+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '6.1-monitoring', 'task2', 'scheduled__2023-09-16T00:00:00+00:00', '--job-id', '489', '--raw', '--subdir', 'DAGS_FOLDER/6.1-monitoring.py', '--cfg-path', '/tmp/tmpvv02lk2b']
[2023-09-18T14:06:26.086+0000] {standard_task_runner.py:85} INFO - Job 489: Subtask task2
[2023-09-18T14:06:26.212+0000] {task_command.py:415} INFO - Running <TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-16T00:00:00+00:00 [running]> on host e891a57bf0af
[2023-09-18T14:06:26.466+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='6.1-monitoring' AIRFLOW_CTX_TASK_ID='task2' AIRFLOW_CTX_EXECUTION_DATE='2023-09-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-16T00:00:00+00:00'
[2023-09-18T14:06:26.481+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-18T14:06:26.488+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "sleep 2 && echo 'Task 2'"]
[2023-09-18T14:06:26.526+0000] {subprocess.py:86} INFO - Output:
[2023-09-18T14:06:28.561+0000] {subprocess.py:93} INFO - Task 2
[2023-09-18T14:06:28.563+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-09-18T14:06:28.618+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=6.1-monitoring, task_id=task2, execution_date=20230916T000000, start_date=20230918T140626, end_date=20230918T140628
[2023-09-18T14:06:28.697+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-18T14:06:28.750+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
